pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_importing-modules-into-pantheon.adoc

```diff
@@ -0,0 +1,19 @@
+[id="importing-modules-into-pantheon__{context}"]
```

Comment:
typo - double underscore before {context}

Reply:
thank you, fixed